src/App.js

```diff
@@ -64,20 +29,41 @@ function App() {
   const updateTokenHandler = (rawToken) => {
     rawToken === undefined ? setToken(null) : setToken(rawToken);
     const url = "http://daily-tasks.test/api/data";
```

Comment:
```suggestion
    const url = apiHost+"/api/data";
```
store `apiHost` in configuration file.